lib/octokit/client/repositories.rb

```diff
@@ -774,6 +774,49 @@ def enable_vulnerability_alerts(repo, options = {})
       def disable_vulnerability_alerts(repo, options = {})
         boolean_from_response(:delete, "#{Repository.path repo}/vulnerability-alerts", options)
       end
+
+      # Check to see if automated security fixes are enabled for a repository
+      #
+      # The authenticated user must have admin access to the repository.
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository.
+      # @return [Boolean] True if automated security fixes are enabled, false otherwise.
+      # @see https://docs.github.com/en/rest/reference/repos#check-if-automated-security-fixes-are-enabled-for-a-repository
+      #
+      # @example
+      #   @client.automated_security_fixes_enabled?("octokit/octokit.rb")
+      def automated_security_fixes_enabled?(repo, options = {})
+        response = get "#{Repository.path repo}/automated-security-fixes", options
```

Comment:
This is not a boolean_from_response capable endpoint; it always returns 200 with an object:

https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#check-if-automated-security-fixes-are-enabled-for-a-repository

Note that unlike the docs, it does *not* return 404 if not enabled, it always returns the object (with enabled: false).